rules/windows/sysmon/sysmon_accessing_winapi_in_powershell_credentials_dumping.yml

```diff
@@ -0,0 +1,24 @@
+title: Accessing WinAPI in PowerShell. Credentials Dumping
+id: 3f07b9d1-2082-4c56-9277-613a621983cc
+description: Detects Accessing to lsass.exe by Powershell
+status: experimental
+author: Natalia Shornikova
```

Comment:
Hello @nsaddler!

Thank you for your contribution!
The rule looks perfect.
Would you mind adding "oscd.community" to the author field?
It will help with the promotion of the initiative and engage more people in the next sprints.

Thank you again!